mkdocs.yml

```diff
@@ -592,6 +638,8 @@ nav:
             - Phonon Dispersions:                       properties-directory/non-scalar/phonon-dispersions.md
             - Phonon Density of States:                 properties-directory/non-scalar/phonon-dos.md
             - Reaction Energy Profile:                  properties-directory/non-scalar/reaction-energy-profile.md
+            # Description of what the new File_Content property is, and what it does
+            - File Content :                            properties-directory/non-scalar/file_content.md
```

Comment:
Let's also add workflow as property here and link to it in explanation

Reply:
Added as a page